astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -1348,3 +1350,13 @@ def check_wcs(header):
     # Check fall back to scale='utc'
     del hdr["TIMESYS"]
     check_wcs(hdr)
+
+
+def test_all_fits_tab():
```

Comment:
Is `all_fits_tab` an appropriate name? Is this what is being tested? Or is it that time has UTC and -TAB in ctype. What is more relevant for this test?

Reply:
I still need to polish this test. I ran out of time the other day.